### PR TITLE
[prometheus-elasticsearch-exporter] Update Docker image to prometheus-community

### PR DESCRIPTION
**Pull request by hierynomus**
#### What this PR does / why we need it:
The Docker image (and documentation) were still pointing to the `justwatch` repository. As work has been migrated to the prometheus-community, these links needed updating and the newly released version is now added to the Helm chart.

#### Which issue this PR fixes
n/a

#### Special notes for your reviewer:
n/a

#### Checklist

- [X] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-elasticsearch-exporter/values.yaml [hunk 1]
```diff
@@ -7,8 +7,8 @@ replicaCount: 1
 restartPolicy: Always
 
 image:
-  repository: justwatch/elasticsearch_exporter
-  tag: 1.1.0
+  repository: quay.io/prometheuscommunity/elasticsearch-exporter
+  tag: 1.2.1
```
Comment (vpartington) [inline charts/prometheus-elasticsearch-exporter/values.yaml:11]: The latest tag is `v1.2.1`, with a `v`.

File: charts/prometheus-elasticsearch-exporter/values.yaml [hunk 2]
```diff
@@ -7,8 +7,8 @@ replicaCount: 1
 restartPolicy: Always
 
 image:
-  repository: justwatch/elasticsearch_exporter
-  tag: 1.1.0
+  repository: quay.io/prometheuscommunity/elasticsearch-exporter
```
Comment (vpartington) [inline charts/prometheus-elasticsearch-exporter/values.yaml:10]: The image name is `prometheuscommunity/elasticsearch-exporter`.

**Review by vpartington — CHANGES_REQUESTED** [review]

File: charts/prometheus-elasticsearch-exporter/values.yaml (hunk 1 above)
Comment (hierynomus) [reply charts/prometheus-elasticsearch-exporter/values.yaml]: Thanks, fixed that!

**Review by monotek — CHANGES_REQUESTED** [review]
Please merge main into your branch to get the "node not ready" problem in ci fixed.
See: https://github.com/prometheus-community/helm-charts/pull/1140

**Comment by hierynomus** [comment]
Superseeded by #1126

**Pull request closed without merging**